common/testutils/functions.go

```diff
@@ -0,0 +1,53 @@
+package testutils
+
+import (
+	"reflect"
+	"time"
+)
+
+func AssertEqual(t Tester, result, expected interface{}) {
```

Comment:
Any `assert` functions should be removed, we have the `testify/assert` package for this.